From 6a76832db224ac5671599ce332717f985a2addc7 Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Thu, 31 Dec 2020 17:16:57 +0100
Subject: utime: Fix compilation error on macOS and Solaris 9 (regr.
 2020-12-24).

Reported by Tom G. Christensen <tgc@jupiterrise.com> in
<https://lists.gnu.org/archive/html/bug-gnulib/2020-12/msg00295.html>.

* lib/utime.c: Include <errno.h>.
---
 ChangeLog   | 7 +++++++
 lib/utime.c | 1 +
 2 files changed, 8 insertions(+)

diff --git a/lib/utime.c b/lib/utime.c
index bf7d7c534..337217950 100644
--- a/lib/utime.c
+++ b/lib/utime.c
@@ -261,6 +261,7 @@ utime (const char *name, const struct utimbuf *ts)
 
 #else
 
+# include <errno.h>
 # include <sys/stat.h>
 # include "filename.h"
 
-- 
cgit v1.2.1

